src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -330,6 +330,14 @@
                 <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
                 <target>Diese internationale Bankleitzahl (BIC) ist nicht mit der IBAN {{ iban }} assoziiert.</target>
             </trans-unit>
+            <trans-unit id="86">
+                <source>This value should be valid JSON.</source>
+                <target>Dieser Wert sollte valides JSON sein.</target>
+            </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>Diese Collection sollte nur eindeutige Elemente enthalten.</target>
```

Comment:
"Eindeutig" translates to "unambiguous" rather than "unique". Correct would be "einzigartig", but that sounds a bit odd.

```suggestion
                <target>Diese Sammlung darf keine doppelten Elemente enthalten.</target>
```